db/migrate/20220720131126_add_emitter_name_field.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddEmitterNameField < ActiveRecord::Migration[6.0]
```

Comment:
Can you remove the previously created migration `20220718142533_modify_emitter_field.rb` and merge with this one if it is still needed ? 

Also, can you rename this migration to match the migration pattern of decidim ? Maybe call the migration as : `add_emitter_to_participatory_processes.decidim-tou.rb`

Reply:
I don't think i should add the ".decidim-tou" behind, it doesn't pass the linter and the other migrations created for this repo does not have it.

Reply:
Alright thanks for the feedback